zh/middleware/modules_main.md

```diff
@@ -0,0 +1,21 @@
+
+# Modules & Commands Reference
+The following pages document the PX4 modules, drivers and commands. They
+describe the provided functionality, high-level implementation overview and how
+to use the command-line interface.
+
+> **Note** **This is auto-generated from the source code** and contains the
```

Comment:
When you translate this bit, you should probably note that the "original" is translated from source and will be up to date, but that this translation may lag slightly.